avm/src/daemon/daemon.rs

```diff
@@ -178,7 +192,92 @@ async fn run_agz_b64(agz_b64: &str) -> DaemonResult<()> {
   Ok(())
 }
 
+fn write_b64_file(file_name: &str, content: &str) -> DaemonResult<()> {
+  let pwd = env::current_dir();
+  match pwd {
+    Ok(pwd) => write(
+      format!("{}/{}", pwd.display(), file_name),
+      base64::decode(content).unwrap(),
+    )?,
+    Err(err) => {
+      let err = format!("{:?}", err);
+      return Err(err.into());
+    }
+  }
+  Ok(())
+}
+
+async fn get_files_b64() -> HashMap<String, String> {
+  let mut files_b64 = HashMap::new();
+  files_b64.insert("Dockerfile".to_string(), get_dockerfile_b64().await);
+  files_b64.insert("app.tar.gz".to_string(), get_app_tar_gz_b64().await);
+  files_b64
+}
+
+fn create_app_tar(content: Option<&String>) -> () {
+  if let Some(content) = content {
+    if let Err(err) = write_b64_file("app.tar.gz", &content) {
+      eprintln!("Could not create app.tar.gz. {:?}", err);
+      std::process::exit(1);
+    }
+  }
+}
+
+async fn generate_token() -> String {
+  let body = json!({
+    "deployConfig": deploy::get_config().await,
+  });
+  post_v1("localDaemonToken", body).await
+}
+
+async fn set_local_daemon_props() -> () {
+  let files_b64 = get_files_b64().await;
+  create_app_tar(files_b64.get(&"app.tar.gz".to_string()));
+  DAEMON_PROPS
+    .set(DaemonProperties {
+      clusterId: "daemon-local-cluster".to_string(),
+      agzB64: get_base_agz_b64(),
```

Comment:
What is this doing here? This shouldn't be here, because that's for `anycloud` not `alan daemon`.

Reply:
How do I run a local cluster that's an actual alan application?

Reply:
Hmm it's here because of https://github.com/alantech/alan/pull/504/files#diff-583f2380a4d780cb04bd93e3492a6f2828052a50e23460f3a2c8d89cf3ade79fR371 but now that you said right now on `main` we are not able to do it?

Reply:
If we're spinning up an `alan daemon` locally (something like `ALAN_TECH_ENV=local alan daemon` it should accept whatever `.agz` file you want, which can be `anycloud.agz` or `my_alan_app.agz`. Perhaps this mode should still spin up the `4142` control port, but have a different endpoint enabled that lets you just pass in the `.agz` file path as a raw string? Eg `curl -k https://localhost:4142/startLocal -d "./my_alan_app.agz"`

Reply:
You're right, I missed that case. What I'll do is giving the possibility to pass an optional argument to `daemon` representing the relative path to the `.agz` file. This way we avoid adding complexity to the process.